src/Inciter/DiagCG.C

```diff
@@ -147,6 +147,9 @@ DiagCG::start()
 {
   auto d = Disc();
 
+  // Start timer measuring time stepping wall clock time
+  d->Timer().zero();
+
```

Comment:
Partly. This resets the timer. (The Timer constructor also starts it from zero, which is what is before this PR.) Starting from the reset point, in `Discretization::status()`, `m_timer.eta()` queries the ETE and ETA, relative to the reset point. So this measures the time elapsed from reset (and in `status`), prints out a status of ETE and ETA.

Before this PR the reset point was mistakenly Discretization's constructor and thus it counted the time including a large part of setup. Since now this will be at the end of `MatCG::setup()` and in `DiagCG::start()`, it will correctly measure only the time-stepping time.

The symptom was that the first time step was measured to be taken orders of magnitude longer than the second one.